src/main/kotlin/racing/domain/Cars.kt

```diff
@@ -0,0 +1,22 @@
+package racing.domain
+
+import racing.domain.strategy.MoveStrategy
+
+class Cars(private val values: List<Car>) : Iterable<Car> {
```

Comment:
이미 잘 아시는 내용이지만, 코틀린은 언어 차원에서 [Delegation](https://kotlinlang.org/docs/delegation.html)을 지원합니다!
이를 활용해보면 다음과 같이 표현할 수 있을 것 같아요.
```suggestion
class Cars(private val values: List<Car>) : List<Car> by value {
```
이렇게 했을 때의 장점은 List에서 제공하는 함수를 Cars는 오버라이딩하지 않아도 사용할 수 있습니다🙂